draft-ietf-quic-recovery.md

```diff
@@ -508,6 +508,12 @@ until it is certain that the server has finished its address validation
 probe timer if the client has not received an acknowledgement for one of its
 Handshake or 1-RTT packets.
 
+When a server receives duplicate CRYPTO data in an Initial packet after sending
+its Initial flight, it can assume the client did not receive all Initial CRYPTO
+data. To speed handshake completion, it SHOULD retransmit all unacknowledged
+Initial CRYPTO data subject to the path validation limits.  After doing so,
+the PTO is re-armed.
+
```

Comment:
Can we formulate duplicate crypto a bit more precise? Does it mean a valid crypto packet with the same packet number as seen before?

Reply:
By CRYPTO data, I meant data sent in CRYPTO frames, which is why I used uppercase for CRYPTO.